tutormfe/templates/mfe/build/mfe/env/development

```diff
@@ -8,5 +8,8 @@ MARKETING_SITE_BASE_URL=http://{{ LMS_HOST }}:8000
 NODE_ENV=development
 REFRESH_ACCESS_TOKEN_ENDPOINT=http://{{ LMS_HOST }}:8000/login_refresh
 STUDIO_BASE_URL=http://{{ CMS_HOST }}:8001
+{% if MFE_COURSE_AUTHORING_MFE_APP and MFE_LEARNING_MFE_APP %}
```

Comment:
Why does adding this setting depend on `MFE_COURSE_AUTHORING_MFE_APP`? Is it because the `LEARNING_BASE_URL` is only used in the authoring MFE?

Reply:
@regisb, that's right.  If you're wondering why I put it here and not in the MFE's own env in the Tutor config, it's because I couldn't readily find a way to access some of the required template variables there.

In any case, once #69 merges this is going into the `[lms|cms]-*-settings` templates.

Reply:
:ok_hand: